### DISCUSSION TITLE
Which planet in our solar system is most often referred to as "🕗⭐"?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#819] [Galaxy Trivia] [answered]
**Bonus:** What is called "Evening Star"?

**Comment by ParanoidUser** [comment]
@Carol42 / @StephDietz, do you happen to know? 😶‍🌫️

**Comment by peterbaccaro** [comment]
The planet most often referred to as the "time star" is **Venus**.
**Venus is sometimes called the "evening star"** or the "morning star" because it is one of the brightest objects in the night sky and can be seen shortly after sunset or before sunrise, depending on its position relative to Earth.
Its appearance as a bright star-like object has led to its association with time and its reference as the "time star."

**Comment by SANKARAMDAS** [comment, marked as answer]
Venus is refer as the "time star"
Similar in structure and size to Earth, Venus's thick atmosphere traps heat in a runaway greenhouse effect, making it the hottest planet in our solar system.
https://solarsystem.nasa.gov/planets/venus/overview/

Venus is the first celestial body to become visible in the sky at the evening and is the last one to disappear from the sky at sunrise. This is why it is known as the Morning and the Evening Sta and Morning Starr.
